Lib/test/libregrtest/refleak.py

```diff
@@ -2,7 +2,6 @@
 import re
 import sys
 import warnings
-from inspect import isabstract
```

Comment:
thanks, victor. I will try your suggestion.

Reply:
> Rather than modifying refleak imports, it would be more generic to only import the refleak module when it's used.
> 
> Currently, clear_caches() is always called. Maybe we can only clear caches when -R 3:3 option is used.

about this idea, I found you and serihy discussed the clear_caches() in https://bugs.python.org/issue23839.
the point is that it's maybe raise some memoryError when calling clear_caches() after -R.
So I thought lazy import isabstract in dash_R() would be more safe(it would be called when we use  -R 3:3 in tests.)